types/applications/filePicker.d.ts

```diff
@@ -0,0 +1,455 @@
+   * Browse files for a certain directory location
+   * @param source  - The source location in which to browse. See FilePicker#sources for details
+   * @param target  - The target within the source location
+   * @param options - Optional arguments
+   *
+   * @returns A Promise which resolves to the directories and files contained in the location
+   */
+  static browse(
+    source: string,
+    target: string,
+    options?: Partial<FilePicker.BrowsingOptions>
+  ): Promise<FilePicker.Result>;
+
+  /**
+   * Configure metadata settings regarding a certain file system path
+   * @param source  - The source location in which to browse. See FilePicker#sources for details
+   * @param target  - The target within the source location
+   * @param options - Optional arguments which modify the request
+   */
+  static configurePath(source: string, target: string, options?: Record<string, unknown>): Promise<unknown>;
+
+  /**
+   * Create a subdirectory within a given source. The requested subdirectory path must not already exist.
+   * @param source  - The source location in which to browse. See FilePicker#sources for details
+   * @param target  - The target within the source location
+   * @param options - Optional arguments which modify the request
+   */
+  static createDirectory(source: string, target: string, options?: Record<string, unknown>): Promise<unknown>;
+
+  /**
+   * General dispatcher method to submit file management commands to the server
+   */
+  protected static _manageFiles(data: FilePicker.ManageData, options?: Record<string, unknown>): Promise<unknown>;
+
+  /**
+   * Dispatch a POST request to the server containing a directory path and a file to upload
+   * @param source  - The data source to which the file should be uploaded
+   * @param path    - The destination path
+   * @param file    - The File object to upload
+   * @param options - Additional file upload options passed as form data
+   * @returns The response object
+   */
+  static upload(
+    source: string,
+    path: string,
+    file: File,
+    options?: Record<string, any>
+  ): Promise<(Response & { path: string; message?: string }) | false | void>;
+
+  /**
+   * Additional actions performed when the file-picker UI is rendered
+   */
+  render(force?: boolean, options?: Application.RenderOptions): this | void | Promise<FilePicker.Result>;
+
+  /**
+   * Activate listeners to handle user interactivity for the FilePicker UI
+   */
+  activateListeners(html: JQuery): void;
+
+  /**
+   * Handle a click event to change the display mode of the File Picker
+   * @param event - The triggering click event
+   */
+  protected _onChangeDisplayMode(event: MouseEvent): void;
+
+  /**
+   * @param event - (unused)
+   * @param event - (unused)
+   * @override
+   */
+  protected _onChangeTab(event: any, tabs: any, active: this['activeSource']): void;
+
+  /**
+   * @param selector - (unused)
+   * @override
+   */
+  protected _canDragStart(selector?: any): boolean;
```

Comment:
I made this any and optional, because it is not used. But I agree making this at least `string | null` like in the parent class is probably better. Thinking more about it, it might not even be a good idea to make a param optional, just because it isn't used.